### PR TITLE
Updates model when changing twitter quota %

### LINKED ISSUE #4684: Twitter user defined % limit not being sent correctly to imports API
### Description
It seems that many times (but not always), no matter what % you choose on the Twitter connector, it sends 80% of your remaining tweets as the maximum.
It seems to happen at least when you choose same day and just some hours of difference, but not sure if there's a correlation or just chance.

### How to reproduce
- Have a user with low Twitter quota, like 10k
- Ttwitter search, same day start/end, just a few hours timespan, 1 term
- select only 1% of remaining quota (so should be 100 tweets on first search, etc.)
- have Chrome developer tools open to see any call to `imports` API performing a `POST`

### Expected results
The POST contains 1% of remaining tweets, something like:
```
user_defined_limits: {twitter_credits_limit: 100}
```

### Actual results
Sometimes it contains 80% of the remaining quota (user had ~9k remaining tweets):
```
user_defined_limits: {twitter_credits_limit: 7761.6}
```

### PR DESCRIPTION
**Pull request by fdansv**
Closes #4684 

@javierarce CR porfi?
cc @Kartones

### CODE REVIEW

**Comment by Kartones** [comment]
Lovely but I have no idea of Backbone :dancer:

File: lib/assets/javascripts/cartodb/common/dialogs/pecan/card.jst.ejs [hunk 1]
```diff
@@ -9,7 +9,11 @@
     <div class="MapCard-contentBody">
       <div class="MapCard-contentBodyRow">
         <h3 class="DefaultTitle">
-          <a href="#" class="DefaultTitle-link u-ellipsLongText" ><%= wizard %> on <strong><%= column %></strong></a>
+          <% if (wizard === "Heatmap") { %>
```
Comment (Kartones) [inline lib/assets/javascripts/cartodb/common/dialogs/pecan/card.jst.ejs]: Is this from this issue?

File: lib/assets/test/spec/cartodb/common/dialogs/create/listing/imports/twitter_import_view.spec.js [hunk 2]
```diff
@@ -142,6 +142,17 @@ describe('common/dialogs/create/imports/twitter_import_view', function() {
...
       expect(this.creditsUsage.$('.js-info').text()).toContain('30%');
     });
+
+    it('should update model when value has changed', function(done) {
+      var UploadModel = require('../../../../../../../../javascripts/cartodb/common/background_polling/models/upload_model.js');
+      expect(this.creditsUsage.$('.js-info').text()).toContain('80%');
+      sinon.spy(UploadModel.prototype, "validate");
+      this.creditsUsage.$(".js-slider").slider('value', 1);
+      setTimeout(function(){
+        expect(UploadModel.prototype.validate.called).toBe(true);
```
Comment (Kartones) [inline lib/assets/test/spec/cartodb/common/dialogs/create/listing/imports/twitter_import_view.spec.js]: And you don't test if the change has also been aplied and now is 1%? :stuck_out_tongue_winking_eye:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5484/))

File: lib/assets/test/spec/cartodb/common/dialogs/create/listing/imports/twitter_import_view.spec.js (hunk 2 above)
Comment (javierarce) [inline lib/assets/test/spec/cartodb/common/dialogs/create/listing/imports/twitter_import_view.spec.js]: +1

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5485/))

**Comment by fdansv** [comment]
@Kartones @javierarce objections fixed. After talking with @javisantana, asynchronous calls with no latency (like 0-time setTimeouts and _.defers) are fine.

**Comment by Kartones** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5486/))

**Comment by javierarce** [comment]
:+1:

**Comment by Cartofante** [comment]
Frontend tests were OK :+1: ([details](http://clinker.cartodb.net/jenkins/job/CartoDB-PR-testing/5488/))

**Pull request merged**